pilot/pkg/model/jwks_resolver.go

```diff
@@ -115,6 +117,17 @@ type jwtKey struct {
 	issuer  string
 }
 
+type CreateFakeJwksFunc func() string
+
+// CreateFakeJwks is a helper function to make a fake jwks when istiod failed to fetch it.
+func CreateFakeJwks() string {
```

Comment:
Why does it need to be dynamic? This is highly costly in the control plane (RSA key is very very expensive) and will change on each xDS (costly in envoy to reload, and drain connections)?

Reply:
make sense, update.